Makefile

```diff
@@ -0,0 +1,11 @@
+.PHONY: all image test
+
+IMAGE_NAME=codeclimate/codeclimate-duplication
+
+all: image test
```

Comment:
Why would I `make all` when `make test` is more intention-revealing and has the same result?